src/main/java/com/launchdarkly/client/StreamProcessor.java

```diff
@@ -227,23 +227,16 @@ public DeleteData() {
   private class DefaultEventSourceCreator implements EventSourceCreator {
     public EventSource createEventSource(EventHandler handler, URI streamUri, ConnectionErrorHandler errorHandler, Headers headers) {
       EventSource.Builder builder = new EventSource.Builder(handler, streamUri)
+          .client(config.httpClient)
```

Comment:
This is overriding some default configuration for okhttp client in the `EventSource`. Those should be copied over.

Reply:
fixed